# SOME DESCRIPTIVE TITLE.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PostgreSQL 9.1 beta 2\n"
"Report-Msgid-Bugs-To: pgsql-bugs@postgresql.org\n"
"POT-Creation-Date: 2013-08-18 12:53+0900\n"
"PO-Revision-Date: 2014-08-16 16:38+0900\n"
"Last-Translator: HOTTA Michihide <hotta@net-newbie.com>\n"
"Language-Team: jpug-doc <jpug-doc@ml.postgresql.jp>\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: fe-auth.c:210 fe-auth.c:429 fe-auth.c:656
msgid "host name must be specified\n"
msgstr "ホスト名を指定しなければなりません\n"

#: fe-auth.c:240
#, c-format
msgid "could not set socket to blocking mode: %s\n"
msgstr "ソケットをブロッキングモードに設定できませんでした: %s\n"

#: fe-auth.c:258 fe-auth.c:262
#, c-format
msgid "Kerberos 5 authentication rejected: %*s\n"
msgstr "Kerberos 5認証が拒絶されました: %*s\n"

#: fe-auth.c:288
#, c-format
#| msgid "could not restore non-blocking mode on socket: %s\n"
msgid "could not restore nonblocking mode on socket: %s\n"
msgstr "ソケットを非ブロッキングモードに戻すことができませんでした: %s\n"

#: fe-auth.c:400
msgid "GSSAPI continuation error"
msgstr "GSSAI続行エラー"

#: fe-auth.c:436
msgid "duplicate GSS authentication request\n"
msgstr "重複するGSS認証要求\n"

#: fe-auth.c:456
msgid "GSSAPI name import error"
msgstr "GSSAPI名のインポートエラー"

#: fe-auth.c:542
msgid "SSPI continuation error"
msgstr "SSPI続行エラー"

#: fe-auth.c:553 fe-auth.c:627 fe-auth.c:662 fe-auth.c:758 fe-connect.c:2005
#: fe-connect.c:3393 fe-connect.c:3611 fe-connect.c:4023 fe-connect.c:4118
#: fe-connect.c:4383 fe-connect.c:4452 fe-connect.c:4469 fe-connect.c:4560
#: fe-connect.c:4910 fe-connect.c:5060 fe-exec.c:3296 fe-exec.c:3461
#: fe-lobj.c:896 fe-protocol2.c:1181 fe-protocol3.c:1544 fe-secure.c:786
#: fe-secure.c:1184
msgid "out of memory\n"
msgstr "メモリ不足です\n"

#: fe-auth.c:642
msgid "could not acquire SSPI credentials"
msgstr "SSPI資格を入手できませんでした"

#: fe-auth.c:733
msgid "SCM_CRED authentication method not supported\n"
msgstr "SCM_CRED認証方式はサポートされていません\n"

#: fe-auth.c:809
msgid "Kerberos 4 authentication not supported\n"
msgstr "Kerberos 4認証はサポートされていません\n"

#: fe-auth.c:825
msgid "Kerberos 5 authentication not supported\n"
msgstr "Kerberos 5認証はサポートされていません\n"

#: fe-auth.c:897
msgid "GSSAPI authentication not supported\n"
msgstr "GSSAPI認証はサポートされていません\n"

#: fe-auth.c:929
msgid "SSPI authentication not supported\n"
msgstr "SSPI認証はサポートされていません\n"

#: fe-auth.c:937
msgid "Crypt authentication not supported\n"
msgstr "Crypt認証はサポートされていません\n"

#: fe-auth.c:964
#, c-format
msgid "authentication method %u not supported\n"
msgstr "認証方式%uはサポートされていません\n"

#: fe-connect.c:798
#, c-format
msgid "invalid sslmode value: \"%s\"\n"
msgstr "sslmodeの値が無効です: \"%s\"\n"

#: fe-connect.c:819
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "SSLサポートが組み込まれていない場合sslmodeの値\"%s\"は無効です\n"

#: fe-connect.c:1023
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "TCPソケットを非遅延モードに設定できませんでした: %s\n"

#: fe-connect.c:1053
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running locally and accepting\n"
"\tconnections on Unix domain socket \"%s\"?\n"
msgstr ""
"サーバに接続できませんでした: %s\n"
"                              ローカルにサーバが稼動していますか?\n"
"                              Unixドメインソケット\"%s\"で通信を受け付けていますか?\n"

#: fe-connect.c:1108
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" (%s) and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"サーバに接続できませんでした: %s\n"
"\tサーバはホスト \"%s\" (%s) で稼動しており、\n"
"\tまた、ポート %s で TCP/IP 接続を受け付けていますか?\n"

#: fe-connect.c:1117
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"サーバに接続できませんでした: %s\n"
"\tサーバはホスト\"%s\"で稼動していますか?\n"
"\tまた、ポート%sでTCP/IP接続を受け付けていますか?\n"

#: fe-connect.c:1168
#, c-format
msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPIDLE)が失敗しました: %s\n"

#: fe-connect.c:1181
#, c-format
msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(TCP_KEEPALIVE)が失敗しました: %s\n"

#: fe-connect.c:1213
#, c-format
msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
msgstr "setsockopt(TCP_KEEPINTVL)が失敗しました: %s\n"

#: fe-connect.c:1245
#, c-format
msgid "setsockopt(TCP_KEEPCNT) failed: %s\n"
msgstr "setsockopt(TCP_KEEPCNT)が失敗しました: %s\n"

#: fe-connect.c:1293
#, c-format
msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
msgstr "WSAIoctl(SIO_KEEPALIVE_VALS)に失敗しました:%ui\n"

#: fe-connect.c:1345
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "無効なポート番号です: \"%s\"\n"

#: fe-connect.c:1378
#, c-format
#| msgid "backup label too long (max %d bytes)"
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)\n"
msgstr "Unixドメインソケットのパス\"%s\"が長すぎます(最大 %d バイト)\n"

#: fe-connect.c:1397
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "ホスト名\"%s\"をアドレスに変換できませんでした: %s\n"

#: fe-connect.c:1401
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr "Unixドメインソケットのパス\"%s\"をアドレスに変換できませんでした: %s\n"

#: fe-connect.c:1606
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "接続状態が無効です。メモリ障害の可能性があります\n"

#: fe-connect.c:1647
#, c-format
msgid "could not create socket: %s\n"
msgstr "ソケットを作成できませんでした: %s\n"

#: fe-connect.c:1669
#, c-format
#| msgid "could not set socket to non-blocking mode: %s\n"
msgid "could not set socket to nonblocking mode: %s\n"
msgstr "ソケットを非ブロッキングモードに設定できませんでした: %s\n"

#: fe-connect.c:1680
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr "ソケットをclose-on-execモードに設定できませんでした: %s\n"

#: fe-connect.c:1699
msgid "keepalives parameter must be an integer\n"
msgstr "keepaliveのパラメータは整数でなければなりません\n"

#: fe-connect.c:1712
#, c-format
msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
msgstr "setsockopt(SO_KEEPALIVE)が失敗しました: %s\n"

#: fe-connect.c:1849
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "ソケットのエラー状態を入手できませんでした: %s\n"

#: fe-connect.c:1883
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "ソケットからクライアントアドレスを入手できませんでした: %s\n"

#: fe-connect.c:1924
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "このプラットフォームでは requirepeer パラメータはサポートされていません\n"

#: fe-connect.c:1927
#, c-format
msgid "could not get peer credentials: %s\n"
msgstr "ピアの資格証明を入手できませんでした: %s\n"

#: fe-connect.c:1937
#, c-format
msgid "local user with ID %d does not exist\n"
msgstr "ID %d を持つローカルユーザは存在しません\n"

#: fe-connect.c:1945
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\"\n"
msgstr "requirepeerは\"%s\"を指定していますが、実際のピア名は\"%s\"です\n"

#: fe-connect.c:1979
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "SSL調停パケットを送信できませんでした: %s\n"

#: fe-connect.c:2018
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "開始パケットを送信できませんでした: %s\n"

#: fe-connect.c:2088
msgid "server does not support SSL, but SSL was required\n"
msgstr "サーバはSSLをサポートしていませんが、SSLが要求されました\n"

#: fe-connect.c:2114
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "SSL調停に対して無効な応答を受信しました: %c\n"

#: fe-connect.c:2189 fe-connect.c:2222
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "サーバからの認証要求を想定していましたが、%cを受信しました\n"

#: fe-connect.c:2389
#, c-format
msgid "out of memory allocating GSSAPI buffer (%d)"
msgstr "GSSAPIバッファの割り当て時のメモリ不足(%d)"

#: fe-connect.c:2474
msgid "unexpected message from server during startup\n"
msgstr "起動時にサーバから想定外のメッセージがありました\n"

#: fe-connect.c:2568
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "接続状態%dが無効です。メモリ障害の可能性があります\n"

#: fe-connect.c:3001 fe-connect.c:3061
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "PGEVT_CONNRESETイベント中にPGEventProc \"%s\"に失敗しました\n"

#: fe-connect.c:3406
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "無効なLDAP URL\"%s\":スキーマはldap://でなければなりません\n"

#: fe-connect.c:3421
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "無効なLDAP URL \"%s\": 区別名がありません\n"

#: fe-connect.c:3432 fe-connect.c:3485
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "無効なLDAP URL \"%s\": 正確に1つの属性を持たなければなりません\n"

#: fe-connect.c:3442 fe-connect.c:3499
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr "無効なLDAP URL \"%s\": 検索スコープ(base/one/sub)を持たなければなりません\n"

#: fe-connect.c:3453
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "無効なLDAP URL \"%s\": フィルタがありません\n"

#: fe-connect.c:3474
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "無効なLDAP URL \"%s\": ポート番号が無効です\n"

#: fe-connect.c:3508
msgid "could not create LDAP structure\n"
msgstr "LDAP構造体を作成できませんでした\n"

#: fe-connect.c:3550
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "LDAPサーバで検索に失敗しました: %s\n"

#: fe-connect.c:3561
msgid "more than one entry found on LDAP lookup\n"
msgstr "LDAP検索結果が複数ありました\n"

#: fe-connect.c:3562 fe-connect.c:3574
msgid "no entry found on LDAP lookup\n"
msgstr "LDAP検索結果が空でした\n"

#: fe-connect.c:3585 fe-connect.c:3598
msgid "attribute has no values on LDAP lookup\n"
msgstr "LDAP検索で属性に値がありませんでした\n"

#: fe-connect.c:3650 fe-connect.c:3669 fe-connect.c:4157
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "接続情報文字列において\"%s\"の後に\"=\"がありませんでした\n"

#: fe-connect.c:3733 fe-connect.c:4337 fe-connect.c:5042
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "接続オプション\"%s\"は無効です\n"

#: fe-connect.c:3749 fe-connect.c:4206
msgid "unterminated quoted string in connection info string\n"
msgstr "接続情報文字列において閉じていない引用符がありました\n"

#: fe-connect.c:3788
msgid "could not get home directory to locate service definition file"
msgstr "サーバ設定ファイルの場所を特定しようとしましたが、ホームディレクトリを取得できませんでした。"

#: fe-connect.c:3821
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "サービス定義\"%s\"がみつかりません\n"

#: fe-connect.c:3844
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "サービスファイル\"%s\"がみつかりません\n"

#: fe-connect.c:3857
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "サービスファイル\"%2$s\"の行%1$dが長すぎます。\n"

#: fe-connect.c:3928 fe-connect.c:3955
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "サービスファイル\"%s\"の行%dに構文エラーがあります\n"

#: fe-connect.c:4570
#, c-format
msgid "invalid URI propagated to internal parser routine: \"%s\"\n"
msgstr "内部パーサ処理へ伝わった無効なURI: \"%s\"\n"

#: fe-connect.c:4640
#, c-format
msgid "end of string reached when looking for matching \"]\" in IPv6 host address in URI: \"%s\"\n"
msgstr "URI \"%s\"内のIPv6ホストアドレスにおいて対応する\"]\"を探している間に文字列が終わりました\n"

#: fe-connect.c:4647
#, c-format
msgid "IPv6 host address may not be empty in URI: \"%s\"\n"
msgstr "URI \"%s\"内のIPv6ホストアドレスが空である可能性があります\n"

#: fe-connect.c:4662
#, c-format
msgid "unexpected character \"%c\" at position %d in URI (expected \":\" or \"/\"): \"%s\"\n"
msgstr "URI（\":\"と\"/\"を除く）内の位置%2$dに想定外の\"%1$c\"文字があります: \"%3$s\"\n"

#: fe-connect.c:4776
#, c-format
msgid "extra key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "URI問い合わせパラメータ内に余分なキーと値を分ける\"=\"があります: \"%s\"\n"

#: fe-connect.c:4796
#, c-format
msgid "missing key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "URI問い合わせパラメータ内にキーと値を分ける\\\"=\\\"がありまません: \"%s\"\n"

#: fe-connect.c:4867
#, c-format
msgid "invalid URI query parameter: \"%s\"\n"
msgstr "無効なURI問い合わせパラメータ:\"%s\"\n"

#: fe-connect.c:4937
#, c-format
msgid "invalid percent-encoded token: \"%s\"\n"
msgstr "無効なパーセント符号化トークン: \"%s\"\n"

#: fe-connect.c:4947
#, c-format
msgid "forbidden value %%00 in percent-encoded value: \"%s\"\n"
msgstr "パーセント符号化された値では%%00値は許されません: \"%s\"\n"

#: fe-connect.c:5270
msgid "connection pointer is NULL\n"
msgstr "接続ポインタはNULLです\n"

#: fe-connect.c:5547
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "WARNING: パスワードファイル\"%s\"がテキストファイルではありません\n"

#: fe-connect.c:5556
#, c-format
msgid "WARNING: password file \"%s\" has group or world access; permissions should be u=rw (0600) or less\n"
msgstr "警告: パスワードファイル \"%s\" がグループメンバもしくは他のユーザから読める状態になっています。この権限はu=rw (0600)以下にすべきです\n"

#: fe-connect.c:5656
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "ファイル\"%s\"からパスワードを読み込みました\n"

#: fe-exec.c:824
msgid "NOTICE"
msgstr "注意"

#: fe-exec.c:1120 fe-exec.c:1178 fe-exec.c:1224
msgid "command string is a null pointer\n"
msgstr "コマンド文字列がヌルポインタです\n"

#: fe-exec.c:1184 fe-exec.c:1230 fe-exec.c:1325
#| msgid "interval(%d) precision must be between %d and %d"
msgid "number of parameters must be between 0 and 65535\n"
msgstr "パラメータ数は0から65535まででなければなりません\n"

#: fe-exec.c:1218 fe-exec.c:1319
msgid "statement name is a null pointer\n"
msgstr "文の名前がヌルポインタです\n"

#: fe-exec.c:1238 fe-exec.c:1402 fe-exec.c:2096 fe-exec.c:2295
msgid "function requires at least protocol version 3.0\n"
msgstr "関数は少なくともプロトコルバージョン3.0が必要です\n"

#: fe-exec.c:1356
msgid "no connection to the server\n"
msgstr "サーバへの接続がありません\n"

#: fe-exec.c:1363
msgid "another command is already in progress\n"
msgstr "他のコマンドを処理しています\n"

#: fe-exec.c:1478
msgid "length must be given for binary parameter\n"
msgstr "バイナリパラメータには長さを指定しなければなりません\n"

#: fe-exec.c:1756
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "想定外のasyncStatus: %d\n"

#: fe-exec.c:1776
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "PGEVT_RESULTCREATEイベント中にPGEventProc \"%s\"に失敗しました\n"

#: fe-exec.c:1906
msgid "COPY terminated by new PQexec"
msgstr "新たなPQexec\"によりCOPYが終了しました"

#: fe-exec.c:1914
msgid "COPY IN state must be terminated first\n"
msgstr "まずCOPY IN状態を終了させなければなりません\n"

#: fe-exec.c:1934
msgid "COPY OUT state must be terminated first\n"
msgstr "まずCOPY OUT状態を終了させなければなりません\n"

#: fe-exec.c:1942
msgid "PQexec not allowed during COPY BOTH\n"
msgstr "COPY BOTH 実行中の PQexec は許可されていません\n"

#: fe-exec.c:2185 fe-exec.c:2252 fe-exec.c:2342 fe-protocol2.c:1327
#: fe-protocol3.c:1683
msgid "no COPY in progress\n"
msgstr "実行中のCOPYはありません\n"

#: fe-exec.c:2534
msgid "connection in wrong state\n"
msgstr "接続状態が異常です\n"

#: fe-exec.c:2565
msgid "invalid ExecStatusType code"
msgstr "ExecStatusTypeコードが無効です"

#: fe-exec.c:2629 fe-exec.c:2652
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "列番号%dは0..%dの範囲を超えています"

#: fe-exec.c:2645
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "行番号%dは0..%dの範囲を超えています"

#: fe-exec.c:2667
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "パラメータ%dは0..%dの範囲を超えています"

#: fe-exec.c:2955
#, c-format
msgid "could not interpret result from server: %s"
msgstr "サーバからの結果を解釈できませんでした: %s"

#: fe-exec.c:3194 fe-exec.c:3278
msgid "incomplete multibyte character\n"
msgstr "不完全なマルチバイト文字\n"

#: fe-lobj.c:155
msgid "cannot determine OID of function lo_truncate\n"
msgstr "lo_truncate関数のOIDを決定できません\n"

#: fe-lobj.c:171
#| msgid "Value exceeds integer range."
msgid "argument of lo_truncate exceeds integer range\n"
msgstr "lo_truncateへの引数が整数範囲を超えています。\n"

#: fe-lobj.c:222
#| msgid "cannot determine OID of function lo_truncate\n"
msgid "cannot determine OID of function lo_truncate64\n"
msgstr "lo_truncate64関数のOIDを決定できません\n"

#: fe-lobj.c:280
#| msgid "Value exceeds integer range."
msgid "argument of lo_read exceeds integer range\n"
msgstr "lo_readへの引数が整数範囲を超えています。\n"

#: fe-lobj.c:335
#| msgid "Value exceeds integer range."
msgid "argument of lo_write exceeds integer range\n"
msgstr "lo_writeへの引数が整数範囲を超えています。\n"

#: fe-lobj.c:426
#| msgid "cannot determine OID of function lo_lseek\n"
msgid "cannot determine OID of function lo_lseek64\n"
msgstr "lo_lseek64関数のOIDを決定できません\n"

#: fe-lobj.c:522
msgid "cannot determine OID of function lo_create\n"
msgstr "lo_create関数のOIDを決定できません\n"

#: fe-lobj.c:601
#| msgid "cannot determine OID of function lo_tell\n"
msgid "cannot determine OID of function lo_tell64\n"
msgstr "lo_tell64関数のOIDを決定できません\n"

#: fe-lobj.c:707 fe-lobj.c:816
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "ファイル\"%s\"をオープンできませんでした: %s\n"

#: fe-lobj.c:762
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "ファイル\"%s\"を読み込めませんでした: %s\n"

#: fe-lobj.c:836 fe-lobj.c:860
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "ファイル\"%s\"に書き込めませんでした: %s\n"

#: fe-lobj.c:947
msgid "query to initialize large object functions did not return data\n"
msgstr "ラージオブジェクト機能を初期化する問い合わせがデータを返しませんでした\n"

#: fe-lobj.c:996
msgid "cannot determine OID of function lo_open\n"
msgstr "lo_open関数のOIDを決定できません\n"

#: fe-lobj.c:1003
msgid "cannot determine OID of function lo_close\n"
msgstr "lo_close関数のOIDを決定できません\n"

#: fe-lobj.c:1010
msgid "cannot determine OID of function lo_creat\n"
msgstr "lo_creat関数のOIDを決定できません\n"

#: fe-lobj.c:1017
msgid "cannot determine OID of function lo_unlink\n"
msgstr "lo_unlink関数のOIDを決定できません\n"

#: fe-lobj.c:1024
msgid "cannot determine OID of function lo_lseek\n"
msgstr "lo_lseek関数のOIDを決定できません\n"

#: fe-lobj.c:1031
msgid "cannot determine OID of function lo_tell\n"
msgstr "lo_tell関数のOIDを決定できません\n"

#: fe-lobj.c:1038
msgid "cannot determine OID of function loread\n"
msgstr "loread関数のOIDを決定できません\n"

#: fe-lobj.c:1045
msgid "cannot determine OID of function lowrite\n"
msgstr "lowrite関数のOIDを決定できません\n"

#: fe-misc.c:295
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "サイズ%luの整数はpqGetIntでサポートされていません"

#: fe-misc.c:331
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "サイズ%luの整数はpqPutIntでサポートされていません"

#: fe-misc.c:610 fe-misc.c:806
msgid "connection not open\n"
msgstr "接続はオープンされていません\n"

#: fe-misc.c:736 fe-secure.c:382 fe-secure.c:462 fe-secure.c:543
#: fe-secure.c:652
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"サーバとの接続が想定外にクローズされました\n"
"        おそらく要求の処理前または処理中にサーバが異常終了\n"
"        したことを意味しています。\n"

#: fe-misc.c:970
msgid "timeout expired\n"
msgstr "タイムアウト期間が過ぎました\n"

#: fe-misc.c:1015
msgid "socket not open\n"
msgstr "ソケットがオープンされていません\n"

#: fe-misc.c:1038
#, c-format
msgid "select() failed: %s\n"
msgstr "select()が失敗しました: %s\n"

#: fe-protocol2.c:91
#, c-format
msgid "invalid setenv state %c, probably indicative of memory corruption\n"
msgstr "setenv状態%cは無効です。メモリ障害の可能性があります\n"

#: fe-protocol2.c:390
#, c-format
msgid "invalid state %c, probably indicative of memory corruption\n"
msgstr "状態%cは無効です。メモリ障害の可能性があります\n"

#: fe-protocol2.c:479 fe-protocol3.c:186
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "待機中にサーバからメッセージ種類0x%02xが届きました"

#: fe-protocol2.c:522
#, c-format
msgid "unexpected character %c following empty query response (\"I\" message)"
msgstr "空の問い合わせ応答(\"I\"メッセージ)の後に想定外の文字%cがありました"

#: fe-protocol2.c:580
#, c-format
msgid "server sent data (\"D\" message) without prior row description (\"T\" message)"
msgstr "サーバが事前の行記述(\"T\"メッセージ)なしにデータ(\"D\"メッセージ)を送信しました"

#: fe-protocol2.c:598
#, c-format
msgid "server sent binary data (\"B\" message) without prior row description (\"T\" message)"
msgstr "サーバが事前の行記述(\"T\"メッセージ)なしにバイナリデータ(\"B\"メッセージ)を送信しました"

#: fe-protocol2.c:618 fe-protocol3.c:385
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "サーバから想定外の応答がありました。受け付けた先頭文字は\"%c\"です\n"

#: fe-protocol2.c:747 fe-protocol2.c:922 fe-protocol3.c:600 fe-protocol3.c:782
msgid "out of memory for query result"
msgstr "問い合わせ結果用のメモリが不足しています"

#: fe-protocol2.c:1370 fe-protocol3.c:1752
#, c-format
msgid "%s"
msgstr "%s"

#: fe-protocol2.c:1382
#, c-format
msgid "lost synchronization with server, resetting connection"
msgstr "サーバとの動機が失われました。接続をリセットしています"

#: fe-protocol2.c:1516 fe-protocol2.c:1548 fe-protocol3.c:1955
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "プロトコルエラー: id=0x%x\n"

#: fe-protocol3.c:341
msgid "server sent data (\"D\" message) without prior row description (\"T\" message)\n"
msgstr "サーバが事前の行記述(\"T\"メッセージ)なしにデータ(\"D\"メッセージ)を送信しました\"\n"

#: fe-protocol3.c:406
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "メッセージの内容がメッセージ種類\"%c\"の長さに合いません\n"

#: fe-protocol3.c:427
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr "サーバとの同期が失われました。受信したメッセージ種類は\"%c\"、長さは%d\n"

#: fe-protocol3.c:478 fe-protocol3.c:518
msgid "insufficient data in \"T\" message"
msgstr "\"T\"メッセージ内のデータが不十分です"

#: fe-protocol3.c:551
msgid "extraneous data in \"T\" message"
msgstr "\"T\"メッセージ内のデータが無関係です"

#: fe-protocol3.c:690 fe-protocol3.c:722 fe-protocol3.c:740
msgid "insufficient data in \"D\" message"
msgstr "\"D\"\"メッセージ内のデータが不十分です"

#: fe-protocol3.c:696
msgid "unexpected field count in \"D\" message"
msgstr "\"D\"メッセージ内のフィールド数が想定外です。"

#: fe-protocol3.c:749
msgid "extraneous data in \"D\" message"
msgstr "”D\"メッセージ内のデータが無関係です"

#. translator: %s represents a digit string
#: fe-protocol3.c:878 fe-protocol3.c:897
#, c-format
msgid " at character %s"
msgstr "(文字位置: %s)"

#: fe-protocol3.c:910
#, c-format
msgid "DETAIL:  %s\n"
msgstr "DETAIL:  %s\n"

#: fe-protocol3.c:913
#, c-format
msgid "HINT:  %s\n"
msgstr "HINT:  %s\n"

#: fe-protocol3.c:916
#, c-format
msgid "QUERY:  %s\n"
msgstr "QUERY:  %s\n"

#: fe-protocol3.c:919
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "CONTEXT:  %s\n"

#: fe-protocol3.c:926
#, c-format
#| msgid "CONTEXT:  %s\n"
msgid "SCHEMA NAME:  %s\n"
msgstr "SCHEMA NAME:  %s\n"

#: fe-protocol3.c:930
#, c-format
#| msgid "DETAIL:  %s\n"
msgid "TABLE NAME:  %s\n"
msgstr "TABLE NAME:  %s\n"

#: fe-protocol3.c:934
#, c-format
#| msgid "CONTEXT:  %s\n"
msgid "COLUMN NAME:  %s\n"
msgstr "COLUMN NAME:  %s\n"

#: fe-protocol3.c:938
#, c-format
msgid "DATATYPE NAME:  %s\n"
msgstr "DATATYPE NAME:  %s\n"

#: fe-protocol3.c:942
#, c-format
#| msgid "CONTEXT:  %s\n"
msgid "CONSTRAINT NAME:  %s\n"
msgstr "CONSTRAINT NAME:  %s\n"

#: fe-protocol3.c:954
msgid "LOCATION:  "
msgstr "LOCATION:  "

#: fe-protocol3.c:956
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:958
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1182
#, c-format
msgid "LINE %d: "
msgstr "行 %d: "

#: fe-protocol3.c:1577
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline: テキストのCOPY OUTを行っていません\n"

#: fe-secure.c:266 fe-secure.c:1121 fe-secure.c:1339
msgid "unable to acquire mutex\n"
msgstr "ミューテックスを獲得できません\n"

#: fe-secure.c:278
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "SSL接続を確立できませんでした: %s\n"

#: fe-secure.c:387 fe-secure.c:548 fe-secure.c:1468
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "SSL SYSCALLエラー: %s\n"

#: fe-secure.c:394 fe-secure.c:555 fe-secure.c:1472
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "SSL SYSCALLエラー: EOFを検知\n"

#: fe-secure.c:405 fe-secure.c:566 fe-secure.c:1481
#, c-format
msgid "SSL error: %s\n"
msgstr "SSLエラー: %s\n"

#: fe-secure.c:420 fe-secure.c:581
msgid "SSL connection has been closed unexpectedly\n"
msgstr "SSL接続が意図せずにクローズされました\n"

#: fe-secure.c:426 fe-secure.c:587 fe-secure.c:1490
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "不明のSSLエラーコード: %d\n"

#: fe-secure.c:470
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "サーバからデータを受信できませんでした: %s\n"

#: fe-secure.c:659
#, c-format
msgid "could not send data to server: %s\n"
msgstr "サーバにデータを送信できませんでした: %s\n"

#: fe-secure.c:779 fe-secure.c:796
msgid "could not get server common name from server certificate\n"
msgstr "サーバ証明書からサーバのコモンネームを取り出すことができませんでした。\n"

#: fe-secure.c:809
msgid "SSL certificate's common name contains embedded null\n"
msgstr "SSL 証明書のコモンネームに null が含まれています\n"

#: fe-secure.c:821
msgid "host name must be specified for a verified SSL connection\n"
msgstr "SSL 接続を検証するためにホスト名を指定しなければなりません\n"

#: fe-secure.c:835
#, c-format
msgid "server common name \"%s\" does not match host name \"%s\"\n"
msgstr "サーバの正式名(common name)\"%s\"がホスト名\"%s\"と一致しません\n"

#: fe-secure.c:970
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "SSLコンテキストを作成できませんでした: %s\n"

#: fe-secure.c:1093
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "証明書ファイル\"%s\"をオープンできませんでした: %s\n"

#: fe-secure.c:1130 fe-secure.c:1145
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "証明書ファイル\"%s\"を読み込めませんでした: %s\n"

#: fe-secure.c:1200
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "SSLエンジン\"%s\"を読み込みできませんでした: %s\n"

#: fe-secure.c:1212
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "SSLエンジン\"%s\"を初期化できませんでした: %s\n"

#: fe-secure.c:1228
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "SSL秘密キーファイル\"%s\"をエンジン\"%s\"から読み取れませんでした: %s\n"

#: fe-secure.c:1242
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "SSL秘密キー\"%s\"をエンジン\"%s\"から読み取れませんでした: %s\n"

#: fe-secure.c:1279
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "証明書はありましたが、秘密キーファイル\"%s\"はありませんでした\n"

#: fe-secure.c:1287
#, c-format
msgid "private key file \"%s\" has group or world access; permissions should be u=rw (0600) or less\n"
msgstr "警告：秘密キーファイル \"%s\" がグループメンバや第三者から読める状態になっています。この権限はu=rw (0600)またはそれ以下とすべきです\n"

#: fe-secure.c:1298
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "秘密キーファイル\"%s\"をロードできませんでした: %s\n"

#: fe-secure.c:1312
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "証明書と秘密キーファイル\"%s\"が一致しません: %s\n"

#: fe-secure.c:1348
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "ルート証明書\"%s\"を読み取れませんでした: %s\n"

#: fe-secure.c:1378
#, c-format
msgid "SSL library does not support CRL certificates (file \"%s\")\n"
msgstr "SSLライブラリがCRL証明書（\"%s\"）をオープンできませんでした\n"

#: fe-secure.c:1411
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file or change sslmode to disable server certificate verification.\n"
msgstr ""
"ルート証明書ファイルを置くためのホームディレクトリが存在しません。\n"
"ファイルを用意するか、サーバ証明書の検証を無効にするように sslmode を変更してください\n"

#: fe-secure.c:1415
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate verification.\n"
msgstr ""
"ルート証明書ファイル\"%s\"が存在しません。\n"
"ファイルを用意するかサーバ証明書の検証を無効にするようにsslmodeを変更してください\n"

#: fe-secure.c:1509
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "証明書を入手できませんでした: %s\n"

#: fe-secure.c:1586
#, c-format
msgid "no SSL error reported"
msgstr "SSLエラーはありませんでした"

#: fe-secure.c:1595
#, c-format
msgid "SSL error code %lu"
msgstr "SSLエラーコード: %lu"

#: win32.c:322
#, c-format
msgid "unrecognized socket error: 0x%08X/%d"
msgstr "不明なソケットエラー 0x%08X/%d"

#~ msgid "verified SSL connections are only supported when connecting to a host name\n"
#~ msgstr "検証されたSSL接続はホスト名への接続時のみサポートされます\n"
